NitroxClient/Communication/Packets/Processors/CraftingProcessor.cs

```diff
@@ -0,0 +1,85 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.Unity.Helper;
+using NitroxModel.Core;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CrafterStartUseProcessor : ClientPacketProcessor<CrafterStartUse>
```

Comment:
What is the reason for moving from Fabricator to Crafter?  Is crafter what is being used internally?  I prefer we stay with fabricator as that is what users will intuitively search for.

Reply:
The processing of these packets relate to both fabricator and workbench. The underlying base logic is the crafter logic from which both are derived. The class that does the magic in nitrox is also named Crafting.